server/files.go

```diff
@@ -407,3 +406,54 @@ func decodeSegmentFileRequest(_ context.Context, r *http.Request) (interface{},
 		requestID: moovhttp.GetRequestID(r),
 	}, nil
 }
+
+type flattenBatchesRequest struct {
+	fileID    string
+	requestID string
+}
+
+type flattenBatchesResponse struct {
+	flattenFileID string `json:"flattenFileID"`
```

Comment:
The OpenAPI states we are returning an `ach.File` object, so we should have that here instead of just the File's ID.